docs/content/guides/operator/exchange-integration.mdx

```diff
@@ -0,0 +1,257 @@
+curl -X POST -H 'Content-type: application/json' --data-raw "$data" $rpc
+```
+
+The response can include a large number of events. Add pagination to the response using the `nextCursor` key in the request. You can determine the corresponding `txDigest` and `eventSeq` from the `id` field of a transaction.
+
+You can add the `txDigest` value instead of the first `null` within the `params`. The second `null` is an integer that defines how many results (up to 1000) to return and the `true` means ascending order. You can use the `nextCursor` so the response starts from a desired point.
+
+The `id` field of any transaction looks like:
+```bash
+"id": {
+         "txDigest": "GZQN9pE3Zr9ZfLzBK1BfVCXtbjx5xKMxPSEKaHDvL3E2",
+         "eventSeq": 6019
+       }
+```
+
+With this data, create a `nextCursor` as follows:
+```bash
+nextCursor : {"txDigest": "GZQN9pE3Zr9ZfLzBK1BfVCXtbjx5xKMxPSEKaHDvL3E2","eventSeq": 6019}
+```
+
+## Blocks vs checkpoints
+
+Sui is a DAG-based blockchain and uses checkpoints for node synchronization and global transaction ordering. Checkpoints differ from blocks in the following ways:
+ - Sui creates checkpoints and adds finalized transactions. Note that transactions are finalized even before they are included in a checkpoint
+ - Checkpoints do not fork, roll back, or reorganize.
+ - Sui creates one checkpoint about every 3 seconds.
+
+### Checkpoint API operations
+
+Sui Checkpoint API operations include:
+ - [sui_getCheckpoint](/sui-api-ref#sui_getCheckpoint) - Retrieves the specified checkpoint.
+ - [sui_getLatestCheckpointSequenceNumber](/sui-api-ref#sui_getLatestCheckpointSequenceNumber) - Retrieves the sequence number of the most recently executed checkpoint.
+ - [sui_getCheckpoints](/sui-api-ref#sui_getCheckpoints) - Retrieves a paginated list of checkpoints that occurred during the specified interval. Pending a future release.
+
+## SUI balance transfer
+
+To transfer a specific amount of SUI between addresses, you need a SUI token object with that specific value. In Sui, everything is an object, including SUI tokens. The amount of SUI in each SUI token object varies. For example, an address could own 3 SUI tokens with different values: one of 0.1 SUI, a second of 1.0 SUI, and a third with 0.005 SUI. The total balance for the address equals the sum of the values of the individual SUI token objects, in this case, 1.105 SUI.
+
+You can merge and split SUI token objects to create token objects with specific values. To create a SUI token worth .6 SUI, split the token worth 1 SUI into two token objects worth .6 SUI and .4 SUI.
+
+To transfer a specific amount of SUI, you need a SUI token worth that specific amount. To get a SUI token with that specific value, you might need to split or merge existing SUI tokens. Sui supports several methods to accomplish this, including some that do not require you to manually split or merge coins.
+
+## Sui API operations for transfers
```

Comment:
This section is quite out of date, as these operations are not long for  this world -- can we add it as a task to revisit this with more modern, PTB or CLI based instructions here?

Reply:
DOCS-112